src/main/java/org/javacord/bot/commands/DocsCommand.java

```diff
@@ -39,14 +39,17 @@ public void onCommand(TextChannel channel, String[] args) {
                         .setThumbnail(getClass().getClassLoader().getResourceAsStream("javacord3_icon.png"), "png")
                         .setColor(Constants.JAVACORD_ORANGE);
                 if (args[0].matches("(classes|class|c)")) { // Search for a class
-                    String searchString = String.join(" ", Arrays.copyOfRange(args, 1, args.length));
-                    populateClasses(channel.getApi(), embed, searchString);
+                    boolean searchAll = args[1].matches("(all|a)");
```

Comment:
Same I wrote about match groups, on-the-fly patterns and switch-case in the other ticket.
Or in this case maybe a `private static final List<String>` and a `.contains(args[1])` as it is not an if-else.